app/models/notifier.rb

```diff
@@ -306,16 +306,20 @@ def get_in_reply_to_message_and_code(email)
   end
 
   def get_site(email)
-    # prefer the to address
-    (Array(email.cc) + Array(email.to)).each do |address|
-      return Site.current if Site.current = Site.where(host: address.downcase.split('@').last).first
+    # prefer the cc address (for reply-all), then the to address
+    to_addresses = Array(email.cc) + Array(email.to)
+    site = nil
+    to_addresses.each do |address|
+      site = Site.where(host: address.downcase.split('@').last).first ||
+             Site.where(email_host: address.downcase.split('@').last).first
```

Comment:
Use 2 (not 7) spaces for indenting an expression in an assignment spanning multiple lines.